src/main/java/Greet.java

```diff
@@ -0,0 +1,30 @@
+import java.util.Scanner;
+
+public class Greet {
```

Comment:
There seems to be 2 main functions, Greet and TaskManager. Maybe can combine both into one to make one of it the main function